features/step_definitions/xmlrpc_common.rb

```diff
@@ -17,6 +17,26 @@
   rabbit = servers[0]
 end
 
+When(/^I call system\.bootstrap\(\) on host "([^"]*)", SSH user name "([^"]*)", \
+password "([^"]*)" and saltSSH "([^"]*)", a new system should be bootstraped\.$/) \
+do |host, ssh_user, ssh_password, salt_ssh_enabled|
```

Comment:
why zou dont use this func:
https://github.com/SUSE/spacewalk-testsuite-base/blob/1bc821bdd3b12ea2443341600e02d5178cd57858/features/support/twopence_init.rb#L101